### PR TITLE
Added test method to check InternalUrl is not valid in creating service.

### PR DESCRIPTION
**Pull request by YAS-SIIN**

### CODE REVIEW

**Review by saeedmaghdam — CHANGES_REQUESTED** [review]
Build fails

File: tests/ZiraLink.Api.Application.UnitTests/Services/ProjectServiceTests.cs [hunk 1]
```diff
@@ -102,6 +103,23 @@ public async Task CreateProject_WhenCustomerIsNotValid_ShouldBeFailed(long custo
...
+            Mock<IBus> mockIBus = new Mock<IBus>();
+            Mock<IHttpTools> mockIHttpTools = new Mock<IHttpTools>();
+            mockIHttpTools.Setup(p => p.CheckDomainExists(It.IsAny<string>())).ReturnsAsync(false);
+
+            ProjectService projectService = new ProjectService(mockILoggerProjectService.Object, TestTools.AppMemoryDbContext, mockIBus.Object, mockIHttpTools.Object);
+
+            var exception = await Assert.ThrowsAsync<ApplicationException>(() => projectService.CreateAsync(customerId, title, domainType, domain, internalUrl, state, CancellationToken.None));
+
+            mockIHttpTools.Verify(p => p.CheckDomainExists(internalUrl), Times.Once());
+            Assert.Equal("Public domain is not allowed", exception.Message);
```
Comment (saeedmaghdam) [inline tests/ZiraLink.Api.Application.UnitTests/Services/ProjectServiceTests.cs:121]: Message could be updated ;)

**Review by saeedmaghdam — CHANGES_REQUESTED** [review]

File: tests/ZiraLink.Api.Application.UnitTests/Services/ProjectServiceTests.cs (hunk 1 above)
Comment (YAS-SIIN) [reply tests/ZiraLink.Api.Application.UnitTests/Services/ProjectServiceTests.cs:121]: Removed.

Comment (saeedmaghdam) [reply tests/ZiraLink.Api.Application.UnitTests/Services/ProjectServiceTests.cs]: It was actually correct! I was just confused. an improvement could be made to the method's name, what you could do is you could change the method's name to something like `CreateProject_WhenInternalUrlIsAccessableFromInternet_ShouldBeFailed` or `CreateProject_WhenInternalUrlIsAPublicUrl_ShouldBeFailed`

**Review by saeedmaghdam — CHANGES_REQUESTED** [review]

**Review by saeedmaghdam — APPROVED** [review]

**Pull request merged**